www/__tests__/enketoHelper.test.ts

```diff
@@ -0,0 +1,245 @@
+import {
+  getInstanceStr,
+  filterByNameAndVersion,
+  resolveTimestamps,
+  resolveLabel,
+  _lazyLoadConfig,
+  loadPreviousResponseForSurvey,
+  saveResponse,
+} from '../js/survey/enketo/enketoHelper';
+import { mockBEMUserCache } from '../__mocks__/cordovaMocks';
+import { mockLogger } from '../__mocks__/globalMocks';
+
+import initializedI18next from '../js/i18nextInit';
+window['i18next'] = initializedI18next;
+
+mockBEMUserCache();
+mockLogger();
+
+global.URL = require('url').URL;
+global.Blob = require('node:buffer').Blob;
```

Comment:
I'm curious about this - why was it needed?
Do Node environments not have these as globals but browser environments do?

Reply:
I remember getting an error about this, I don't remember exactly what the error was, but I found that the best way to fix it was to set it like this. [comment about this](https://github.com/e-mission/e-mission-phone/pull/1063#issuecomment-1762249637). I think the issue had something to do with the testing environemnt